packages/account-export/src/lib/helpers/importAccount.spec.ts

```diff
@@ -0,0 +1,36 @@
+import type { AccountImportData } from "@near-wallet-selector/core";
```

Comment:
Do you think there would be value in testing invalid/unexpected inputs here? E.g. invalid accounts JSON, empty `secretKey` values, etc.

Reply:
validity of accounts are actually get filtered during account selection step. 
(Invalid accounts will become un-selectable) 

Will update encryption and decryption logic with exception handling